.github/workflows/check-coverage.yml

```
@@ -0,0 +1,22 @@
+on: ["push", "pull_request"]
```

```suggestion
on:
  push:
     branches:  ["master"]
  pull_request:
     branches: ["master"]
```

to avoid spurious push triggers